src/lib/helpers/exchangeRefreshForAccessToken.js

```diff
@@ -0,0 +1,35 @@
+export const exchangeRefreshForAccessToken = ({ clientId, clientSecret, tokenEndpoint, fetch = window.fetch, token }) => {
+  const payload = {
+    client_secret: clientSecret,
+    client_id: clientId,
+    grant_type: "refresh_token",
+    scope: "openid, profile",
+    refresh_token: token.refresh_token
```

Comment:
Some auth providers don't provide refresh tokens. Before setting a timer to use the refresh token, we must check if there actually is a refresh token.